### PR TITLE
add preocurse

### PR DESCRIPTION
**Pull request by anderMartin92**

### CODE REVIEW

File: staff/ander/precourse/airlines/index-pro.js [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+    while(semaforo==false || user=="") {
+        alert("NO ESCRIBAS NUMEROS!")
+        user= prompt("Como te llamas?");
+        soloLetras(user);
+    }
+    console.log("SKYLAB AIRLINES")
+    bienve=("Bienvenido " + user);
+    console.log(bienve);
+    //lINEA ESTETICA (no queda muy bien);
+    for (let index = 0; index < bienve.length; index++) {
```
Comment (manuelbarzi) [inline staff/ander/precourse/airlines/index-pro.js:55]: do not use `let`, but `var` instead (we are working is ES5)

File: staff/ander/precourse/airlines/index.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    return semaforo=false;
+
+    }
+}
+}
+
+
+
+function vuelos(){
+for(n in flights){
```
Comment (manuelbarzi) [inline staff/ander/precourse/airlines/index.js:66]: declare `n`, use `var`

File: staff/ander/precourse/bingo/index.js [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+            lin1=false;
+        }
+        if (carton[i].valor!="X" && carton[i].fila==2) {
+            lin2=false;
+        }
+        if (carton[i].valor!="X" && carton[i].fila==3) {
+            lin3=false;
+        }
+    }
+    if (lin1==true && linea1==false){
```
Comment (LLorch1984) [inline staff/ander/precourse/bingo/index.js:154]: try to don't use == on boleean results if ( x === true) its the same than if(x) or the negative if(!) means false

Comment (LLorch1984) [reply staff/ander/precourse/bingo/index.js:154]: i(!x)

Comment (anderMartin92) [reply staff/ander/precourse/bingo/index.js:154]: ok, i will change

Comment (anderMartin92) [reply staff/ander/precourse/bingo/index.js:154]: can't change line for if (lin1) && (!linea1)){

File: staff/ander/precourse/airlines/index.js (hunk 2 above)
Comment (manuelbarzi) [reply staff/ander/precourse/airlines/index.js:67]: no need to declare `n` outside, but just inline `for (var n ...`

**Pull request merged**